test/tests/benchmark/utils/time_logger.py

```diff
@@ -0,0 +1,61 @@
+import os
+import time
+import json
+
+"""
+Class to manage time logger file
+"""
+
+class timeLogger(object):
```

Comment:
I read logging manual and tried on my side, and it turns out it contains many customized operations for benchmark. I think it would be better to keep it inside benchmark/. 

The case info file created by timeLogger would be:
```
{
    "time marker": {
        "start": "2016/03/25 05:13:36",
        "end": "2016/03/25 05:13:45"
    },
    "interval": 1,
    "case name": "poller",
    "log path": "20160325-051329"
}
```
The reasons not to move it to logger is:
1. logging.handler, formatter and filter cannot write the log in this format.
I did experiments and the log can only be one json object per log item, and cannot be piled as one json object. Though changing benchmark parser to handle a different format is viable, the efforts exceed that in combining timeLogger to modules/logger.
2, write_interval(), write_log_path() and others are heavily customized for benchmark, and some of them are executed when initializing the log file. It might not be suitable to put in modules/logger as common methods.